template/package.json

```diff
@@ -3,14 +3,13 @@
   "version": "1.0.0",
   "description": "My Moleculer-based microservices project",
   "scripts": {
-    "build": "tsc --build tsconfig.json",
```

Comment:
It's missing.

Reply:
I changed the dockerfile to use the build in production, the way it was really not working